src/Service/DocumentUploader.php

```diff
@@ -102,21 +102,22 @@ public function handleViewDocument(Document $document, bool $forceDownload = fal
             stream_copy_to_stream($fileStream, $outputStream);
         });
 
-        // We use document name as filename.
-        $filename = $document->getDocumentName();
+        // We use document name as filename - it may not contain / or \, so we replace those.
+        $documentName = $document->getDocumentName();
+        $ext = pathinfo($document->getFilename(), PATHINFO_EXTENSION);
 
-        $slugger = new AsciiSlugger();
-        $fallbackFilename = $slugger->slug($document->getDocumentName())->__toString();
+        $documentName = str_replace(['/', '\\'], '-', $documentName);
```

Comment:
You may want to replace `%` as well since it's not allowed in the fallback filename.

Reply:
Escaped `%`.